x-pack/plugins/apm/public/components/shared/Stacktrace/index.tsx

```diff
@@ -0,0 +1,182 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiLink, EuiTitle } from '@elastic/eui';
+import { get, isEmpty } from 'lodash';
+import React, { PureComponent } from 'react';
+import styled from 'styled-components';
+import { Stackframe } from '../../../../typings/APMDoc';
+import { px, units } from '../../../style/variables';
+import { CodePreview } from '../../shared/CodePreview';
+import { EmptyMessage } from '../../shared/EmptyMessage';
+// @ts-ignore
+import { Ellipsis } from '../../shared/Icons';
+import { FrameHeading } from './FrameHeading';
+
+export interface StackframeCollapsed extends Stackframe {
+  libraryFrame?: boolean;
```

Comment:
`Stackframe` already has `libraryFrame` - with snake case:
https://github.com/elastic/kibana/blob/04a37c465400cd1a80f709f2b311171de057f227/x-pack/plugins/apm/typings/APMDoc.ts#L64

We do some camelCasing here (that we should stop doing):
https://github.com/elastic/kibana/blob/04a37c465400cd1a80f709f2b311171de057f227/x-pack/plugins/apm/public/services/rest/callApi.ts#L36

Reply:
I'd suggest we disable the camelCasing for the spans endpoint:
https://github.com/elastic/kibana/blob/abeafa1ae14c59a7c658b4192ea9686d2fa0549a/x-pack/plugins/apm/public/services/rest/apm/transactions.ts#L14

Like we did for the transactions endpoint:
https://github.com/elastic/kibana/blob/abeafa1ae14c59a7c658b4192ea9686d2fa0549a/x-pack/plugins/apm/public/services/rest/apm/transactions.ts#L56-L58

Eventually we can remove the camel casing altogether (I think spans (this) and errors are the only hold-ups).

Reply:
i started down this path, but it ended up being a larger change than expected, since there are so many references to the camel cased properties. I'm not sure if i want to add that to this pr to create a new issue for it?

Reply:
Okay, that makes sense. A new PR is probably a good idea. This PR lgtm.